PowerShell/ScubaGear/baselines/aad.md

```diff
@@ -203,7 +203,7 @@ Figure 1: Depiction of MFA methods from weakest to strongest. _Adapted from [Mic
 #### MS.AAD.3.1v1
 Phishing-resistant MFA SHALL be enforced for all users.
 
-The phishing-resistant methods **Microsoft Entra ID certificate-based authentication (CBA)**, **FIDO2 Security Key** and **Windows Hello for Business** are the recommended authentication options since they offer forms of MFA with the least weaknesses. For federal agencies, Microsoft Entra ID CBA supports federal PIV card authentication directly to Microsoft Entra ID.
+The phishing-resistant methods **Microsoft Entra ID certificate-based authentication (CBA)**, **FIDO2 Security Key**, **Windows Hello for Business**, and **device-bound passkeys** (in the Microsoft Authenticator app on iOS and Android) are the recommended authentication options since they offer forms of MFA with the least weaknesses. For federal agencies, Microsoft Entra ID CBA supports federal PIV card authentication directly to Microsoft Entra ID.
```

Comment:
Explicitly saying use of the Microsoft Authenticator app may be seen as an endorsement. I recommend we opt for general language here:

"and **device-bound passkeys** (in the authenticator app of choice) are the recommended authentication options..."

Reply:
makes sense, will update!

Reply:
The generic wording makes a lot of sense here, looks good.